test/e2e/pivoting.go

```diff
@@ -33,6 +33,7 @@ const (
 	ironicBasicAuth            = "IRONIC_BASIC_AUTH"
 	Kind                       = "kind"
 	NamePrefix                 = "NAMEPREFIX"
+	NameSuffix                 = "NAMESUFFIX"
```

Comment:
This is not used.
```suggestion
```

Reply:
That is different. It is defined in the struct here
https://github.com/metal3-io/cluster-api-provider-metal3/blob/762326f049f3c16d89797b3fce5ab762d406a77e/test/e2e/pivoting.go#L295

This is a constant that is never referenced. The other constants above are used to get the value from the e2e config as in 
https://github.com/metal3-io/cluster-api-provider-metal3/blob/762326f049f3c16d89797b3fce5ab762d406a77e/test/e2e/pivoting.go#L95
But this `NameSuffix` does not exist in the e2e config so we never take it from there.

Reply:
true, good catch! I will remove it, with other change